js/jquery-comments.js

```diff
@@ -594,13 +594,23 @@
         },
 
         updateToggleAllButton: function(parentEl) {
+            // Don't hide replies if maxRepliesVisible has negative value
+            if (this.options.maxRepliesVisible < 0) return;
+
             var childCommentsEl = parentEl.find('.child-comments');
             var childComments = childCommentsEl.find('.comment');
             var toggleAllButton = childCommentsEl.find('li.toggle-all');
             childComments.removeClass('hidden-reply');
 
+            // Assign all child comments to be hidden
+            var hiddenReplies = childComments;
```

Comment:
This would be easier to read in form:

if (this.options.maxRepliesVisible == 0) {
	var hiddenReplies = childComments;
} else {
	var hiddenReplies = childComments.slice(0, -this.options.maxRepliesVisible);
}